### PR TITLE
Change controller bindings to command style, make score commands better

### PR DESCRIPTION
**Pull request by nobody5050**

### CODE REVIEW

File: src/main/cpp/RobotContainer.cpp [hunk 1]
```diff
@@ -88,29 +88,21 @@ void RobotContainer::ConfigureButtonBindings() {
...
 
 #ifndef TEST_SWERVE_BOT
-
-    m_leftClimb.SetDefaultCommand(ExtendClimbCommand(
+    m_driverController.LeftTrigger(0.01).WhileTrue(ExtendClimbCommand(
         &m_leftClimb, [this] { return -m_driverController.GetLeftTriggerAxis(); },
         [this] { return 0; }).ToPtr());
 
-    m_rightClimb.SetDefaultCommand(ExtendClimbCommand(
+    m_driverController.RightTrigger(0.01).WhileTrue(ExtendClimbCommand(
```
Comment (having11) [inline src/main/cpp/RobotContainer.cpp:95]: Make this our deadzone constant

File: src/main/cpp/RobotContainer.cpp [hunk 2]
```diff
@@ -88,29 +88,21 @@ void RobotContainer::ConfigureButtonBindings() {
 
 
 #ifndef TEST_SWERVE_BOT
-
-    m_leftClimb.SetDefaultCommand(ExtendClimbCommand(
+    m_driverController.LeftTrigger(0.01).WhileTrue(ExtendClimbCommand(
```
Comment (having11) [inline src/main/cpp/RobotContainer.cpp:91]: Same comment about deadzone constant

File: src/main/include/commands/ScoreAmpCommand.h [hunk 3]
```diff
@@ -1,68 +1,34 @@
...
-            m_score->StartScoringRamp(ScoringDirection::AmpSide);
-            isFinished = false;
+#include "BaseScoreCommand.h"
+
+class ScoreAmp : public BaseScoreCommand {
+ public:
+  explicit ScoreAmp(ScoringSubsystem* subsystem, IntakeSubsystem* intk)
+      : BaseScoreCommand(subsystem, intk, ScoringDirection::AmpSide) {}
+
+  void Execute() override {
```
Comment (having11) [inline src/main/include/commands/ScoreAmpCommand.h:10]: Why can't `Execute` be moved into the base?

Comment (nobody5050) [reply src/main/include/commands/ScoreAmpCommand.h:10]: It can be, I’ll fix that

**Comment by having11** [comment]
Are we adding the ScoreAmpSpeaker command too?

File: src/main/cpp/subsystems/ScoringSubsystem.cpp [hunk 4]
```diff
@@ -58,10 +59,19 @@ void ScoringSubsystem::SpinSpeaker() {
     m_speakerUpperSpinnyBoi.Set(ScoringConstants::kSpeakerUpperSpeed);
 }
 
+void ScoringSubsystem::SpinSubwoofer() {
```
Comment (having11) [inline src/main/cpp/subsystems/ScoringSubsystem.cpp:62]: The upper-level `SpinVectorSide`, `StartScoringRamp`, `GetMotorAtScoringSpeed`, and `GetMotorFreeWheel` were never updated to reflect the addition of `ScoringDirection::Subwoofer`

**Review by having11 — CHANGES_REQUESTED** [review]

**Review by having11 — APPROVED** [review]

**Pull request merged**